servicebroker_node/test/fat/index_test.js

```diff
@@ -0,0 +1,34 @@
+var supertest = require("supertest");
```

Comment:
in my understanding,  if we invoke a function directly and verify the input/output , that is unit test.  If we launch the server (here, service broker app) , and invoke the RESTful API to verify the request/response, it is function test. if more apps are launched together, it is integration test.   So, here, we start service broker app , and call /v2/catalog endpoint, it should be a function test.  Is that right?